### PR TITLE
change some words selections in zh

### PR DESCRIPTION
**Pull request by huangbow**
Eliminate potential discriminative words , as "他" === he/him, "它" === it. "他将虚拟DOM与DOM本身区别开" === "He can tell the difference between Virtual DOM and DOM itself" and it's needed to correct.

### CODE REVIEW

**Comment by developit** [comment]
Sounds like a good change @huangbow, hopefully @lcxfs1991 can review! ❤️

File: content/lang/zh/index.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ toc: false
     </p>
 
     <p> 
-		Preact也是web平台的一等公民，他将虚拟DOM与DOM本身区别开，注册真实的事件处理函数，很好地与其它库一起工作。
```
Comment (lcxfs1991) [inline content/lang/zh/index.md]: 原翻译没问题

Comment (lcxfs1991) [inline content/lang/zh/index.md]: 只有虚拟dom进行diff吧，有diff再更新dom

**Comment by huangbow** [comment]
Thanks for the review. The original content is "It diffs Virtual DOM against the DOM itself". I would say the existing one is so abscure.

**Comment by lcxfs1991** [comment]
It's my understanding that react-like framework would diff virtual dom first. It won't diff virtual dom and its corresponding real dom.  So I guess there are two options may fit author's meaning.

1. preact diffs virtual dom instead of it's corresponding dom.
2. preact diffs virtual dom for it's corresponding dom.

what's your opinion? @developit

**Comment by developit** [comment]
@lcxfs1991 I'm reading this second option as "preact diffs virtual dom against it's corresponding dom" - is that correct? If so, that seems more accurate than the first.

**Comment by lcxfs1991** [comment]
Gotcha. Let me fix the translation and merge the request.

**Comment by huangbow** [comment]
The Diff algorithm is the true treasure of Preact. Thanks guys for helping me make it more clear.

**Comment by huangbow** [comment]
I'm hesitate to take the merged translates. @developit Can you be more explicit about "preact diffs virtual dom against it's corresponding dom"? I'm afraid there's a gap between new translate and your idea. So what does Preact really diff:
1) vdom vs. vdom
2) vdom vs. corresponding dom

**Comment by huangbow** [comment]
@lcxfs1991 I would be happy to read [diff code](https://github.com/developit/preact/blob/master/src/vdom/diff.js) start from line 34.

**Comment by developit** [comment]
@huangbow I appreciate the concern :)
Preact definitely falls into option #2 in your question - it diffs DOM <-> VDOM.

**Comment by huangbow** [comment]
@developit My original proposal (PR) was reflected what option 2. But unfortunately, @lcxfs1991 "corrected" it as vdom vs. vdom, which made me feel anxious. I hope @lcxfs1991 can stick to what I originally pulled. Thanks guys.

**Comment by hassanbazzi** [comment]
Any updates on this?

**Pull request closed without merging**